src/components/layout.js

```diff
@@ -69,16 +69,7 @@ const Layout = ({ location, title, children }) => {
           <Excalidraw />
         </div>
         <span style={{ clear: "both" }} />
-        {location.pathname === rootPath ? (
-          <h1
-            style={{
-              ...scale(1.2),
-              marginTop: 0,
-            }}
-          >
-            {title}
-          </h1>
-        ) : null}
+        {location.pathname === rootPath ? <h1>{title}</h1> : null}
```

Comment:
👍 on making the title a little smaller. 👎 on adding back the full margin. I feel like it leaves too much whitespace:

![image](https://user-images.githubusercontent.com/25517624/78042577-2ee3be80-7340-11ea-88fe-615b01a4cb07.png)

Reply:
I think that is totally all right.. it gives some space from the `Open Excalidraw`.. 

Makes more sense here: https://excalidraw-blog-git-homepage.excalidraw-team.now.sh/building-excalidraw-p2p-collaboration-feature/

Reply:
And the header doesn't jump from homepage to post..

Reply:
Maybe just remove the space on the home page?

Reply:
Then there is a jump when clicking on a post..